lindera-ipadic-builder/src/lib.rs

```diff
@@ -94,6 +94,9 @@ impl<'a> CSVRow<'a> {
 
     fn from_line_user_dict(line: &'a String) -> CSVRow<'a> {
         let fields: Vec<_> = line.split(",").collect();
+        if fields.len() >= 13 {
```

Comment:
Great work!
One suggestion/question about the wrong CSV format.

I think we should notify errors if the length of columns is wrong. e.g. 12
Because the user forgot the 13th column, then Lindera creates the CSVRow as a simple user dictionary.
That is unexpected behavior.

日本語でも。
CSVのカラムの数のチェックをしたほうがいいかと思います。
もし、12カラムしかない行があった場合、simple user dictionaryとして扱おうとします。
カラムの情報がずれておかしなことになるかと。

Reply:
確認ありがとうございます。
カラム数をチェックするようにしました。